index.js

```diff
@@ -297,321 +334,345 @@ var color_lookup =
+    "b" : 1,
+    "name" : "blue"
   },
   {
-    "h" : 45,
-    "s" : 1,
-    "b" :0.5,
-    "name" : "dark turquoise"
+    "h" : 70,
+    "s" : 0,
+    "b" : 1,
+    "name" : "lavender"
   },
   {
-    "h" : 40,
+    "h" : 70,
     "s" : 0.5,
-    "b" :0.5,
-    "name" : "dark green"
-  },
-  {
-    "h" : 50,
-    "s" : 1,
-    "b" :0.5,
-    "name" : "dark cyan"
+    "b" : 0.5,
+    "name" : "navy blue"
   },
   {
-    "h" : 50,
+    "h" : 70,
     "s" : 0.5,
-    "b" :0.5,
-    "name" : "dark cyan"
+    "b" : 1,
+    "name" : "lavender blue"
   },
   {
-    "h" : 55,
+    "h" : 70,
     "s" : 1,
-    "b" :0.5,
-    "name" : "dark blue"
+    "b" : 0.5,
+    "name" : "dark navy blue"
   },
   {
-    "h" : 60,
+    "h" : 70,
     "s" : 1,
-    "b" :0.5,
-    "name" : "navy blue"
+    "b" : 1,
+    "name" : "blue"
   },
   {
-    "h" : 65,
+    "h" : 75,
     "s" : 0.5,
-    "b" :0.5,
-    "name" : "navy blue"
+    "b" : 1,
+    "name" : "lavender"
   },
   {
-    "h" : 70,
-    "s" : 0.5,
+    "h" : 75,
+    "s" : 1,
     "b" : 0.5,
-    "name" : "navy blue"
+    "name" : "dark purple"
   },
   {
-    "h" : 65,
+    "h" : 75,
     "s" : 1,
-    "b" :0.5,
-    "name" : "dark navy blue"
+    "b" : 1,
+    "name" : "purple"
   },
   {
-    "h" : 70,
-    "s" : 1,
-    "b" :0.5,
-    "name" : "dark navy blue"
+    "h" : 80,
+    "s" : 0.5,
+    "b" : 1,
+    "name" : "pinkish purple"
   },
-  {
-    "h" : 75,
+   {
+    "h" : 80,
     "s" : 1,
-    "b" :0.5,
+    "b" : 0.5,
     "name" : "dark purple"
   },
   {
     "h" : 80,
     "s" : 1,
-    "b" :0.5,
-    "name" : "dark purple"
+    "b" : 1,
+    "name" : "purple"
+  },
+  {
+    "h" : 85,
+    "s" : 0,
+    "b" : 1,
+    "name" : "light pink"
+  },
+  {
+    "h" : 85,
+    "s" : 0.5,
+    "b" : 0.5,
+    "name" : "purple"
   },
   {
+    "h" : 85,
+    "s" : 0.5,
+    "b" : 1,
+    "name" : "light fuchsia"
+  },
+   {
     "h" : 85,
     "s" : 1,
-    "b" :0.5,
+    "b" : 0.5,
     "name" : "dark fuchsia"
   },
+  {
+    "h" : 85,
+    "s" : 1,
+    "b" : 1,
+    "name" : "fuchsia"
+  },
   {
     "h" : 90,
     "s" : 0.5,
-    "b" :0.5,
+    "b" : 0.5,
     "name" : "dark fuchsia"
   },
+  {
+    "h" : 90,
+    "s" : 0.5,
+    "b" : 1,
+    "name" : "hot pink"
+  },
   {
     "h" : 90,
     "s" : 1,
-    "b" :1,
+    "b" : 0.5,
     "name" : "dark fuchsia"
   },
   {
     "h" : 90,
     "s" : 1,
-    "b" :0.5,
-    "name" : "dark fuchsia"
+    "b" : 1,
+    "name" : "fuchsia"
   },
   {
     "h" : 95,
-    "s" : 1,
-    "b" :0.5,
-    "name" : "dark magenta"
+    "s" : 0,
+    "b" : 1,
+    "name" : "pink"
   },
   {
-    "h" : 90,
+    "h" : 95,
     "s" : 0.5,
-    "b" :1,
-    "name" : "hot pink"
+    "b" : 1,
+    "name" : "light pink"
   },
   {
-    "h" : 0,
-    "s" : 0,
-    "b" :0,
-    "name" : "black"
+    "h" : 95,
+    "s" : 1,
+    "b" : 0.5,
+    "name" : "dark magenta"
+  },
+  {
+    "h" : 95,
+    "s" : 1,
+    "b" : 1,
+    "name" : "magenta"
   },
```

Comment:
Question - is each value of "h" supposed to have 6 values of (s,b) ?

Reply:
it could. right now it including all the values of html colors so not all of the h values have 6 values of (s,b). I could work on making sure all of them have 6. lmk

Reply:
got it. Let's make that an issue and work in it. But we can merge this for now?

Reply:
yes! will make it an issue.

Reply:
I've made the changes and created new issue.